src/data/MetaDataItem.py

```diff
@@ -46,9 +61,16 @@ def to_json(self) -> dict:
             'collision_type': self.collision_type,
             'description': self.description,
             'location': self.location,
-            'accident_indexes': self.accident_indexes,
             'is_cancelled': self.is_cancelled,
-            'tags': self.tags
+            'tags': self.tags,
+            'bb_frames': self.bb_frames,
+            'bb_ids': self.bb_ids,
+            'bb_clss': self.bb_clss,
+            'bb_x1s': self.bb_x1s,
```

Comment:
instead of having multiple lists for x1,y1,x2,y2, could we have just one list of 4-tuples (x1,x2,y,y2) or a list of dictionaries? I think it would make the code cleaner